model/custom_file.go

```diff
@@ -25,3 +29,37 @@ func (c CustomFile) GzippedBase64Content() string {
 	}
 	return out
 }
+
+func (c CustomFile) RenderContentWithEncoding(gzb64 bool, ctx interface{}) (string, error) {
```

Comment:
I prefer making this `RenderGzippedBase64Content(ctx interface{})` because it tells about the specific encoding of gzip+base64 than `RenderContentWithEncoding(true, ...)`. The `gzb64 bool` can be omitted because we use it from nowhere right now?

Reply:
That makes sense, please see this amended version with separate RenderGzippedBase64Content function as suggested (keeping RenderContent for use where encoding is not desired).